pkg/rtc/participant.go

```diff
@@ -1518,7 +1518,7 @@ func (p *ParticipantImpl) hasPendingMigratedTrack() bool {
 }
 
 func (p *ParticipantImpl) onUpTrackManagerClose() {
-	close(p.rtcpCh)
+	p.rtcpCh <- nil
```

Comment:
Is this another way to close a channel?

Reply:
Is the problem that the check in `sfu.Receiver` happened (`if w.closed.Load()`) and then the channel closed and then we posted. Life with atomics :-)

Reply:
yeah, receiver already closed, but buffer has rtcp feedback in OpsQueue that is executing and passed the close check. So don't close channel here, just send a nil into channel to notify participant's rtcpSendWorker to exit